cwiczenie3/osoba.php

```diff
@@ -63,21 +63,21 @@ public function roznicaWieku(Osoba $osoba)
 
 interface Pojazdy
 {
-   public function sredniaPredkosc();
+   public function sredniaPredkosc($pp1, $ps);
```

Comment:
To jest klasa która zwraca średnią prędkość. A nie ją ustawia. Ustawianie masz albo ręczne wewnątrz każdej metody albo przez kontruktor.